deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -78,49 +62,19 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - description: Represents a Ceph cluster.
-      displayName: Ceph Cluster
-      kind: CephCluster
```

Comment:
Instead of removing, we can instead change the display names and the descriptions to make it clear that these resources are not to be created or managed by users.

The console sorts the list of CRDs based on the display name, so we could possibly add a prefix that would ensure that the prefixed CRDs are shown later. And we could mention in the description of each that they are not be managed manually.